### DISCUSSION TITLE
Latest SDL commit breaks build -  SDL_SystemCursor: rename enum items to match CSS.

### DISCUSSION THREAD

**Discussion by adrian-purser** [ocornut/imgui#7652] [New Users Build/Link/Run issues ONLY!]
Hi,

SDL_SYSTEM_CURSOR_ definitions have been renamed in the latest commit of the SDL repo.  These definitions are used in the SDL3 backend and therefore it no longer builds. Probably affects the SDL2 backend but I haven't tried it.

The affected file is backends/imgui_impl_sdl3.cpp

Thanks

**Comment by ocornut** [comment]
Fixed by f8de9fec8c5dc72fca6c35fc6052a8bb9813f5fc. Feel free to open an issue if there's breakage in dear imgui backends itself, SDL3 is currently very frequently changing..
